packages/core/content-releases/admin/src/services/utils.ts

```diff
@@ -2,6 +2,9 @@ import { getFetchClient } from '@strapi/helper-plugin';
 
 import type { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
 
+/* -------------------------------------------------------------------------------------------------
+ * Axios data
+ * -----------------------------------------------------------------------------------------------*/
 export interface QueryArguments<TSend> {
```

Comment:
I find it a bit odd to mix inline exports and end-of-file exports in the same file

Reply:
Good catch, looks like it doesn't even need to be exported